fe/lib/components/QuestionnaireBuilder/FormBuilder/components/NewQuestionForm/NewQuestionForm.tsx

```diff
@@ -146,13 +154,18 @@ export default ({ hideForm, initialValues }: NewQuestionFormProps) => {
               id={field.name}
               label="Type"
               message={errors.questionType?.message}
+              onChange={(event) =>
+                field.onChange(event.currentTarget.value as QuestionTypeValue)
```

Comment:
Why does this one need a cast?

Reply:
A string union of the option values is expected, but the Braid dropdown component does not provide a strongly typed `event.currentTarget.value`